src/Xamarin.Android.Build.Tasks/Utilities/Files.cs

```diff
@@ -198,16 +198,16 @@ public static void ExtractAll(ZipArchive zip, string destination, Action<int, in
 			int i = 0;
 			int total = zip.Entries.Count;
 			foreach (var entry in zip.Entries) {
-
-				if (string.Equals(entry.FullName, "__MACOSX", StringComparison.OrdinalIgnoreCase) ||
-						string.Equals(entry.FullName, ".DS_Store", StringComparison.OrdinalIgnoreCase))
+				if (entry.FullName.Contains ("__MACOSX") ||
```

Comment:
I don't understand how the previous check was "bad", and I'm not fond of the new check as it's locale-sensitive.

The commit message mentions that files named `_MACOSX` weren't ignored, but this new check *still* won't support that, as `__` != `_` -- the `.Contains()` check is looking for *two* underscores, not one, just as it originally did, so `_MACOSX` will *still* be processed.

Or is the problem that `entry.FullName` contains directory information, so it could be `foo/bar/_MACOSX`? If so, then the `_MACOSX` check should presumably follow the `.DS_Store` check and use `.EndsWith()` with `StringComparison.OrdinalIgnoreCase`.